dappnode_package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "arbitrum-nitro.public.dappnode.eth",
-  "version": "1.1.0",
+  "version": "1.0.11",
   "description": "Arbitrum Nitro Node",
   "type": "service",
   "author": "0xAtum",
   "contributors": [
     "0xAtum (https://github.com/0xAtum)",
     "Voss <voss@visnovalabs.io> (https://github.com/alexpeterson91)"
   ],
-  "upstreamVersion": "v2.3.3-6a1c1a7",
+  "upstreamVersion": "v2.3.4",
```

Comment:
```suggestion
  "upstreamVersion": "v2.3.4-b4cc111",
```

Reply:
I have CI setup for this repo and have been trying to get in touch with you for a while now, pinged on twitter as requested but never got a DM back.  Ill take over maintinence my fork has full integration with the dappmanager dashboard, chain status bar, setup wizard to use global envs to set EL and CL automatically, or choose custom entry (i.e. `http://geth.dappnode:8545`) and handle any initializations needed or not.  
But in order to keep this particular package (this ENS, as opposed to `arbitrum-nitro.dnp.dappnode.eth`) i need you to give me permission to publish the repo `arbitrum-nitro.public.dappnode.eth` so users of this branch can simply upgrade to versions i publish going forward.  If you don't give me permission on the publishing key, (its not hard nor urgent atm, but i can walk you through it) then ill have to publish a new package, likely as a DNP not a public package and any user of this is gonna have to remove this package and install the new one.  Up side is as a DNP itll be whitelisted into the store when published, downside being a remove and replace arbitrum nitro package public for DNP.  

One thing to be careful with regardless is versions, you MUST use Semver, and you shouldn't match release versions with the upstream release, let the bot handle versioning for you, because you have releases out of order and stuff and dont just bump using the CI which here suggests the next semantic version from the last published release/marked as latest release in GH, because it will always increment by 1 as a patch release which is why we start with v0.1.0 major version 0 since its the first, minor version 1 and finally last digit is patch version 0 since there's not been one before.  so it should just go patch by patch intil theres something major that changes everythniing, migration, volume erase, new build etc. just have it go as it suggests.

Reply:
Hmm strange I didn't receive any DM from you on my end, might have dismissed it accidentally. Sorry

Yeah  I got lazy and stop using the Semver, my bad.

Poking you back on Twitter.